bitnami/grafana-operator/templates/deployment.yaml

```diff
@@ -54,10 +49,10 @@ spec:
       {{- else }}
       affinity:
         {{- if not (empty .Values.operator.podAffinityPreset) }}
-        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.operator.podAffinityPreset "component" "grafana-operator" "context" $) | nindent 10 }}
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.operator.podAffinityPreset "customLabels" $podLabels "context" $) | nindent 10 }}
         {{- end }}
         {{- if not (empty .Values.operator.podAntiAffinityPreset) }}
-        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.operator.podAntiAffinityPreset "component" "grafana-operator" "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.operator.podAntiAffinityPreset "customLabels" $podLabels "context" $) | nindent 10 }}
```

Comment:
Just to make sure, is it expected that we remove the old `component` `grafana-operator`  entry?

Reply:
Yes it's expected, because as you can see at `spec.template.metadata.labels` we're not adding the `app.kubernetes.io/component` label.